assets/css/dashboard/pending-screens-page.scss

```diff
@@ -1,5 +1,11 @@
 .pending-screens-page {
   height: 100%;
+
+  .add-new-button {
+    height: 48px;
+    background-color: #8ecdff;
+    color: #212529;
```

Comment:
Do we have design tokens for things like this? (i.e. variables)

Reply:
We did not for this color and we should have. I double checked designs and this color is showing as primary in Figma. This is not what `$button-primary` was set to though so I renamed the current `$button-primary` to `$button-secondary` (which is how we were using it anyway from what I can tell) and added this color as `$button-primary`.

Reply:
I'm a little confused; from the commit, it looks like you meant _this_ color (`212529`) should be "secondary", and the one above (`8ecdff`) should be "primary". It also looks like this instance of `212529` is still here, rather than being updated to use the "secondary" variable.

Reply:
Woops, sorry about that. Forgot to do one more Find All before committing the change.